metrics/report/report_dockerfile/parallel.R

```diff
@@ -114,11 +111,14 @@ if ( skip_points == 0 ) {
 
 # See https://www.r-bloggers.com/ggplot2-easy-way-to-mix-multiple-graphs-on-the-same-page/ for
 # excellent examples
-master_plot = grid.arrange(
+# Explicitly suppress warnings during the generation of the plot, as geom_smooth can generate a mass
```

Comment:
I'm wondering if suppressing warnings here is a good idea? We can just acknowledge that on runs of < 10 we'll get a warning, or we can remove the smoothing. Just afraid of what else it might mask.

Reply:
Agreed. I contemplated it briefly. Sadly there seems no trivial way to calculate if/when warnings will get spat out, and the only clue I could find to some sort of boundary was '1000 points', which I think is more when it changes algorithm rather than spit out warnings.
Let's pick a number and I'll add the if/else. I'm thinking >20 data points maybe? (at the point where the view starts to get dense, and a smoothed line might be useful).